browser.js

```diff
@@ -74,7 +76,7 @@ function pino (opts) {
     levels,
     timestamp: getTimeFunction(opts)
   }
-  logger.levels = pino.levels
+  logger.levels = getLevels(opts)
```

Comment:
I did it this way to avoid mutating `pino.levels`, which is public, to avoid a possible breaking change.

Or should I mutate `pino.levels` to add custom levels there anyway?

Reply:
this is correct